build.sbt

```diff
@@ -85,8 +94,7 @@ def coulombParserDeps = Seq(
 )
 
 lazy val coulomb_parser = (project in file("coulomb-parser"))
```

Comment:
Some projects remain jvm only

Reply:
Do you have any thoughts on why coulomb-parser isn't working with scala.js?  My prediction was that if you could get fully-qualified unit type names, then QuantityParser should work

Reply:
QuantityParser calls the reflect api and I didn't want to dig further as I was mostly trying to get the core and units to run.

Also the QuantityParser tests were dependent on pureconfig which is not available in scala.js but I think one test could be extracted to run in both

Reply:
possibly it doesn't have a lot of application in scala JS.  I don't have a strong feel for it.  If it's easy to make work, I guess I'm in favor of doing it, but OTOH if it is hard then we can leave it for some other time.